setup.py

```diff
@@ -126,23 +269,28 @@ def copy_data_tree(datadir, destdir):
 
 # Conditionally copy the GDAL data. To be used in conjunction with
 # the bdist_wheel command to make self-contained binary wheels.
-if os.environ.get('PACKAGE_DATA'):
```

Comment:
It is important to keep this code block conditionally enabled as it was done previously. The reason is that `PACKAGE_DATA` is used for building wheels as the content is copied to the wheels. Otherwise, the content should not be copied. Examples include when GDAL is installed on the system or using conda.

Reply:
Much appreciated.